### ISSUE TITLE
Broken NormalizationLayer and GroupNormLayer

### ISSUE THREAD

**Issue by oKatanaaa** [MakiResearchTeam/MakiFlow#33]
`mf.layers`
**NormalizationLayer and GroupNormLayer**
Wrong calculation of the running mean and running variance. In current implementation these are dependent on the batch size what is wrong. Example: 
`self.running_mean = np.zeros((N, 1, 1, self.G, 1))`
`self.running_variance = np.ones((N, 1, 1, self.G, 1))`
Calculated mean and variance must be averaged along the batch dimension. Example of such an implementation:
https://github.com/Bingohong/GroupNormalization-tensorflow-keras/blob/master/group_normalization.py
